src/Microsoft.DotNet.Compiler.Common/Executable.cs

```diff
@@ -53,6 +53,23 @@ public void MakeCompilationOutputRunnable()
             ExportRuntimeAssets();
         }
 
+        private void VerifyCoreclrPresenceInPackageGraph()
```

Comment:
I feel that Coreclr is the right thing here, but don't actually know, what do you think?